docs/qa/qa-test-plan.md

```diff
@@ -0,0 +1,332 @@
+# QA Test Plan: React Transcript Editor
+
+Last updated: _3rd Jan 2018_     
+version: _>=0.2.0_
+
+This doc provides a lightweight set of steps and checklists for manual QA test. This should be done before every major release.
+
+_The assumption underlying this doc is that anyone, even without technical skills, should be able to conduct QA testing._
+
+## Overview
+React Transcript Editor it's a react component, to make transcribing audio and video easier and faster. Please see project repository the Github [README](https://github.com/bbc/react-transcript-editor/blob/master/README.md) page,for an overview of what the component does and how it works (https://github.com/bbc/react-transcript-editor). 
+
+<!-- Here is a video demo of the  main use case: https://youtu.be/4z143-nJlzs.  -->
+
+## Where to test:
+<!-- _URL of where testing should be performed (staging, sandbox)_ -->
+
+The app can be tested using the demo application at 
+https://bbc.github.io/react-transcript-editor/
+
+For updating the the demo app with latest version of the component [see these instructions](https://github.com/bbc/react-transcript-editor#build---demo)
+
+## Where to log bugs: 
+<!-- _Provide link to Fogbugz, Github, Trello, etc. Also include to whom the bugs should be assigned (if applicable)._ -->
+
+If you find any bugs or issues, please [open an issue in Github](https://github.com/bbc/react-transcript-editor/issues/new?template=bug_report.md) and assign it to [@pietrop](https://github.com/pietrop).
+
+If it's connect to one or more of the QA steps listed below make a note of the corresponding number.
+
+For things like typos feel free to directly do a PR with the changes.
+
+## Browsers/devices: 
+By default we aim for the component to work on the latest major version of the following browsers on the below devices.
+
+Desktop (Mac, Win and Linux), Tablet (iOS, Android, Win), Mobile (iOS, Android, Win), on Chrome, Firefox, Safari, Internet Explorer and Edge.
```

Comment:
ah, yes fair point, that was lifted from another QA doc, what's your pick?

eg we could divide into core and nice to have, eg core could be
- [ ] Desktop - Mac  - Chrome 
- [ ] Desktop - Windows - Chrome
- [ ] Desktop - Windows - Internet Explore (Because of BBC users)

Not part of core but feedback welcome on the rest?